README.md

```diff
@@ -40,9 +50,10 @@ This template comes with a basic project structure that you may find useful:
 | [`src/dev_tools.rs`](./src/dev_tools.rs) | Dev tools for dev builds                              |
 
 Feel free to move things around however you want, though.
+The best way to get started is to just play around with what you find in [`src/game/`](./src/game).
 
 > [!Tip]
-> Be sure to check out the amazing [3rd-party tools](./docs/tooling.md) in the Bevy ecosystem!
+> Be sure to check out our [recommended 3rd-party tools](./docs/tooling.md)!
```

Comment:
This does match the header better, but we _are_ also linking to bevy assets in there.

Also I know this is obviously not the intention, but "our recommended 3rd-party tools" may make it sound like we wrote them.

Reply:
Oh, good catch. I'll rephrase.

Reply:
[`fcf41a2` (#158)](https://github.com/TheBevyFlock/bevy_quickstart/pull/158/commits/fcf41a249130122f730d58a1752ac5287d89a362)